introduction.md

```diff
@@ -7,3 +7,11 @@ I'm the bestest git ninja out there!
 I slice and dice the repo and put it all together without dropping a hash.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?